matrix-helpers.el

```diff
@@ -159,5 +159,158 @@ PAIRS is a spliced plist."
            when value
            collect (cons key value)))
 
+;;;; FIXME: Argh
+
+(defun matrix--url-http-create-request (&optional ref-url)
```

Comment:
I would at least stick a comment here saying we pulled it from url.el, otherwise it can get a bit confusing :)